x-pack/test/api_integration/apis/endpoint/resolver.ts

```diff
@@ -41,25 +41,6 @@ const expectLifecycleNodeInMap = (node: LifecycleNode, nodeMap: Map<string, Tree
   compareArrays(genNode!.lifecycle, node.lifecycle, true);
 };
 
-/**
```

Comment:
No longer need this helper because the origin node is now included in the tree's ancestry map